zebROS_ws/src/field_obj_tracker/include/field_obj_tracker/convert_coords.h

```diff
@@ -10,7 +10,7 @@ class ConvertCoords
 	image_geometry::PinholeCameraModel model_;
 
 public:
-	ConvertCoords(const image_geometry::PinholeCameraModel &model);
+	ConvertCoords(const sensor_msgs::CameraInfo camera_info_);
```

Comment:
Looks like this is left over from previous work, which is cool, we want to get it merged also

One improvement here is to make this a const reference, add & before camera_info_. That'll eliminate the need for making a copy of the CameraInfo object when passing it into the function.

Reply:
Oh, and a nitpick, but make the function argument just camera_info.  We try to reserve variables ending in _ for class member vars.